app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -88,6 +88,12 @@ $padding-xs-horizontal: 0.25rem;
 $nav-link-padding: 0.5rem 1rem;
 $grid-float-breakpoint: 768px;
 
+// popovers
+$popover-bg: $stanford-black;
+.popover-body {
+  color: #fff!important;
```

Comment:
can we use `$popover-body-color` here?  Why do we need `!important`?

Reply:
Yeah this happened because the stackoverflow I found did not include that variable. I was looking for it everywhere except the source code which is where I should have gone first. Thank you that was driving me crazy.